app/models/friendship.rb

```diff
@@ -0,0 +1,12 @@
+class Friendship < ApplicationRecord
+  belongs_to :user
+  belongs_to :friend, class_name: 'User'
+  scope :pending_requests, -> { where(status: false) }
+
```

Comment:
When a friendship is confirmed, an additional row with reversed user-friend pair is created in the friendship model

```
# Friendship model

def confirm_friend
  self.update_attributes(confirmed: true)
  Friendship.create!(friend_id: self.user_id,
                  user_id: self.friend_id,
                  confirmed: true)
end

```